base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -188,11 +235,12 @@ Result<StatusFetcherOutput> StatusFetcher::FetchOneInstanceStatus(
   auto response = ResponseFromSiginfo(infop);
   if (response.status().code() != cvd::Status::OK) {
     instance.set_state(cvd::INSTANCE_STATE_UNREACHABLE);
-    instance_manager_.UpdateInstance(group, instance);
     instance_status_json["warning"] = "cvd status failed";
-    instance_status_json["instance_name"] = instance.name();
-    instance_status_json["status"] = HumanFriendlyStateName(instance.state());
+  } else {
+    CF_EXPECT(UpdateInstanceWithStatusResult(instance, instance_status_json));
```

Comment:
nit: can the `if` be the positive conditional, i.e.
```
if (v1 == v2) {
  ...
} else {
  ...
}
```
rather than
```
if (v1 != v2) {
  ...
} else {
  ...
}
```
See [go/tott/685](http://go/tott/685)

Reply:
Done